classes/classes/Player.as

```diff
@@ -763,8 +763,12 @@ use namespace kGAMECLASS;
 			{
 				if (hasDragonWingsAndFire())
 					race = isBasilisk() ? "dracolisk" : "dragonewt";
-				else
-					race = isBasilisk() ? "basilisk"  : "lizan";
+				else {
+					if (lowerBody == LOWER_BODY_TYPE_HARPY && hasScales() && InCollection(wingType, WING_TYPE_DRACONIC_LARGE, WING_TYPE_HARPY, WING_TYPE_FEATHERED_LARGE, WING_TYPE_BAT_LIKE_LARGE))
+						race = "cockatrice"; // should also have avian face, but we have no such thing for now
+					else
+						race = isBasilisk() ? "basilisk"  : "lizan";
+				}
```

Comment:
Hopefully very soon:
https://github.com/Kitteh6660/Corruption-of-Champions-Mod/compare/master...Stadler76:Cockatrices
Main gdoc (finished, unless I missed something):
https://docs.google.com/document/d/1sCjCrpp4or8kp02OcdLs5CP5bZYXXjFy8qOp8PPyTT4/edit
Cockatrice TF item - Ton o’ Trice (stubby implementation started):
https://docs.google.com/document/d/1WnOBXDVLGAryHolTOq2DY1eS7GU0LmfolQ4wS972KDo/edit